packages/ckeditor5-image/src/imageinsert/imageinsertviaurlui.ts

```diff
@@ -182,9 +182,7 @@ export default class ImageInsertViaUrlUI extends Plugin {
 
 		dialog.show( {
 			id: 'insertImageViaUrl',
-			title: this._imageInsertUI.isImageSelected ?
-				t( 'Update image URL' ) :
-				t( 'Insert image via URL' ),
+			title: t( 'Image via URL' ),
```

Comment:
The description in `contexts.json` chould be updated, now it look like this:

```
"Image via URL": "The label for the insert image via url menu bar button (standalone button).",
```

Especially the `menu bar`.

Reply:
Fixed https://github.com/ckeditor/ckeditor5/pull/17560/commits/9b162b1542c8b7bb25be41c86f170e55e8fd46db. I kept `menu bar`, as it's defined the same way in several other places.